core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.ts

```diff
@@ -94,6 +98,12 @@ export class OpportunitiesListItemComponent {
         'background-color': this.opportunity.labelColor
       };
     }
+    if (this.opportunity && this.opportunity.translationLengthInWords) {
+      this.translationLengthLabel = (
+        this.opportunity.translationLengthInWords < 20 ? 'short' : 'long');
```

Comment:
Hm are we sure we even want to label "long" translations? It might enough to just highlight the short ones. Also since translations near the cutoff don't really seem "long", e.g. 20 words, so the label doesn't seem like a strict dichotomy.

Reply:
I think just having short label won't be of much good here ? That might be confusing as some cards are labelled as short and some are not labeled at all. WDYT ?

Well there has to be some cutoff ? Also it was mentioned in PRD that 20 is the cutoff to determine whether the translation is short or long, that's why I went with it.

Reply:
I am not sure if I totally agree with this. @seanlip WDYT, should we have labels only for short translations or both ?

Reply:
@sagangwee is right, only short translations need to be labelled. These are just to help reviewers identify "quick wins". If you label everything, the short ones stand out less.

Reply:
Thanks, I have updated accordingly.